src/Pages/Cart/CartComponents/CartCard/CartCard.jsx

```diff
@@ -0,0 +1,85 @@
+import { Loader } from "../../../../Components";
+import { useCartContext } from "../../../../context/Cart.context";
+import { useFilterReducerContext } from "../../../../context/FilterReducer.context";
+import "./CartCard.css";
+
+export const CartCard = () => {
+  const { loading, dispatch } = useFilterReducerContext();
+  const { cartProducts, removeFromCart, changeCartQty } = useCartContext();
+
+  return (
+    <>
+      {loading ? (
+        <Loader />
+      ) : (
+        <>
+          {cartProducts.map((products) => (
```

Comment:
You can remove, `{` and `<>` here. As, I have suggested in the last PR.

Reply:
Yes its been taken care of for some reason its not in this PR i will fix it